testdata/project-v3/api/v1/webhook_suite_test.go

```diff
@@ -114,7 +114,7 @@ var _ = BeforeSuite(func() {
 	err = (&Captain{}).SetupWebhookWithManager(mgr)
 	Expect(err).NotTo(HaveOccurred())
```

Comment:
It is scaffolded here, right?

Reply:
I think you have not run make generate before pushing the change. I cloned your repo and generated a project with that, it doesn't generate that code. Also, I tried to run the `make generate` locally and it is throwing same error. Could you please try that?